net-p2p/cardano-db-sync/pkg-message

```diff
@@ -0,0 +1,51 @@
+[
+{ type: install
+  message: <<EOM
+/!\ This FreeBSD package REQUIRES postgresql DB to be initialized   /!\
+/!\ as per the following example for the software to function       /!\
+
+If starting with a fresh postgresql install:
+service postgresql initdb
+service postgresql start
+
+To create a new database, this example uses "root" username for Cardano mainnet.
+For a testnet configuration, write .pgpass file to "testnet-configs" instead.
+
+su postgres
+psql
+
+CREATE ROLE root SUPERUSER LOGIN;
+ALTER USER root PASSWORD 'YourPasswordHere';
+
+CREATE DATABASE cexplorer WITH OWNER = root ENCODING 'UTF8' TEMPLATE template0;
+ALTER SCHEMA public OWNER TO root;
+
+\q
+
+exit
+
+setenv PGPASSFILE /var/db/cardano_db_sync/mainnet-configs/.pgpass
+echo "/tmp/:5432:cexplorer:*:*" > $PGPASSFILE
```

Comment:
What does that `/tmp/:5432:cexplorer:*:*` mean? I was expecting to see something like `yourpassword` here.

Reply:
'yourpassword' is only if you want to access the db over tcp/ip to manage DB remotely. While you are logged into the same account as specified in the db initialization, you can access the db without using password. This is the point, as you do not want the db-sync binary having to enter a password to run commands. I followed this here: https://github.com/input-output-hk/cardano-db-sync/blob/master/config/pgpass-mainnet

Except, that socket is not available on BSD/Darwin, it's under /tmp/ instead.

Reply:
All right, but what `5432` means in this string?

It is also strange that pgsql puts its sockets into `/tmp` ._.

Reply:
5432 is the port. even tho the access is done via local socket , for whatever reason, 5432 is part of the socket name so this directive is used by the client to know what to append to the socket name to connect ( it's like... /tmp/PGSQL.5432 or something like that ) i'm guessing this is how they differentiate the many different instances you can run under same server but using different ports?

If you have complaints as to why postgresql makes a socket under /tmp/ in BSD and Darwin but normal location in Linux, i suggest you take it up with their developers ;-)